.buildkite/filebeat/scripts/package-step.sh

```diff
@@ -18,7 +18,7 @@ if are_files_changed "$changeset"; then
       - label: ":ubuntu: Packaging Linux X86"
         key: "package-linux-x86"
         env:
-          PLATFORMS: "+all linux/amd64 linux/arm64 windows/amd64 darwin/amd64 darwin/arm64"
```

Comment:
We publish a darwin/aarach64 image for these Beats, are we sure removing it is right? 

* https://artifacts.elastic.co/downloads/beats/filebeat/filebeat-8.12.1-darwin-aarch64.tar.gz
* https://artifacts.elastic.co/downloads/beats/auditbeat/auditbeat-8.12.1-darwin-aarch64.tar.gz
* https://artifacts.elastic.co/downloads/beats/heartbeat/heartbeat-8.12.1-darwin-aarch64.tar.gz

Is this right?

Reply:
let me check one more time the root cause of the problem

Reply:
It looks like the darwin/arm64 platform isn't supported by only in 7.17 branch.
@cmacknz Please correct me if I'm wrong.
If it's correct, I'll recreate PR for 7.17 only
All mentioned in the issue description builds are related to 7.17.

Reply:
That's correct, see the below details:

The current packaging pipeline for `7.17` is not building `arm64`:
- https://github.com/elastic/beats/blob/6b671dd901b6b1824d12d33568859af2a5937402/.ci/packaging.groovy#L306-L325

As far as I see `darwin/arm64` was added to `>= 8.2` as per https://github.com/elastic/beats/pull/29585

https://www.elastic.co/downloads/past-releases/elastic-agent-8-1-3 vs https://www.elastic.co/downloads/past-releases/elastic-agent-8-2-0

and no arm64 artifacts in https://www.elastic.co/downloads/past-releases/elastic-agent-7-17-18

Reply:
I've created a new one https://github.com/elastic/beats/pull/38098